core/metacling/src/TCling.cxx

```diff
@@ -2824,6 +2824,17 @@ Bool_t TCling::IsLibraryLoaded(const char* libname) const
    return s_IsLibraryLoaded(libname, fInterpreter);
 }
 
+Bool_t TCling::HasPCMForLibrary(const char *libname) const
```

Comment:
It seems that this will be accurate only if the library and pcm existed at process startup time.   What if the library was created later (for example via ACliC)? 

Also, we need to add at least one line of doxygen documentation describing the function.

Reply:
PCMs are only created at compilation time. So when a new library was created, this function will always return false and it's a correct behavior.

Reply:
>  When a new library was created by ACLiC and this function was called, it'll always return false which is a correct behavior.

Why?  The compilation steps done by ACLiC may have produced a pcm file.   Why can't we use it then?

Reply:
As far as I know, we're not supporting that feature and I'm not sure if we should. @vgvassilev ?

Reply:
I talked with Vassil, and true we need to be able to support lazy loading of pcm generated by ACLiC. I'll add FIXME here.

Reply:
One of the design goal (and actual use) of ACLiC is for it to be a simple way to produce a complete 'ROOT' library including the necessary infrastructure (i.e. dictionary etc.) to be used 'instead' of 'complex' makefile/CMakeLists.txt for the simple cases.   So unless there is really a heavy (development) cost to support creating 'full fledge' library we should try (or least keep in mind how we could).

Reply:
Yeah I agree, I'll try to generate pcm for aclic and load them when it exists instead of rootmaps.